lib/dictionary.rb

```
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+class Dictionary
+  def initialize
+    @content = {}
+  end
+
+  def definition(word)
+    content[word]&.flatten
+  end
+
+  def add(word, definition)
+    raise 'Cannot store empty string' if word.empty?
```

I would raise a dedicated error object rather than a plain string.